### ISSUE TITLE
Ecowitt Addon

### ISSUE THREAD

**Issue by ahgray** [tlskinneriv/awnet_local#41]
Sorry if this is the wrong place to ask. But If I have a WS-5000 weather station as well as an Ecowitt GW1000 gateway would you say that this integration would be the currently best way of getting local access to the data, seeing as how the official Ecowitt integration is now broken since it cant use SSL connections.

**Comment by tlskinneriv** [comment]
Does the device have a way to push data to a custom server? If so, what protocols (Ambient Weather, Wunderground, etc.) can it send with?

This integration and add-on combo currently supports Ambient Weather style data updates from devices which can be configured to upload to a custom server.

**Comment by ahgray** [comment]
Sorry, I kinda assumed you were already familiar with it. Prior to the recent update to the ambient weather firmware allowing for your add-on, the only way I was able to find to get local import to home assistant was through the Ecowitt GW1100 Wi-Fi Weather Station Sensor Gateway. Which essentially read the 915mhz data and acted as a gateway. There is a HACS integration https://github.com/PoppyPop/homeassistant_ecowitt that worked well, but then HA created their own Core integration for the ecowit data, however the ecowitt device doesn't support HTTP so you had to expose all of HA over http. The HACS integration sets up its own separate listener port similar to your setup. 

Long story short I incorrectly assumed that this was based upon the ecowitt hacs integration or that you were aware of it, and that by using this one I may be able to cut out the middleman(the GW1100).

**Comment by tlskinneriv** [comment]
Were you able to get this to work? From my understanding, the  WS-5000 should be able to send data directly to this add-on with the correct configuration. 

For the Ecowitt, we may be able to modify the code to support the protocol it uses as well since it seems quite similar.

**Comment by tlskinneriv** [comment]
Closing because stale.
